app/core/cache.py

```
@@ -31,4 +35,37 @@ def cache_key_builder(func: Callable[..., Any], namespace: str = "default", *, r
+            redis = aioredis.from_url(
+                settings.REDIS_URL,
+                encoding="utf-8",
+                decode_responses=True,
+                socket_connect_timeout=3
+            )
+            # Test connection
+            await redis.ping()
+            backend = RedisBackend(redis)
+            cache_type = "Redis"
+            logger.info("✅ Successfully connected to Redis cache")
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)